src/plugins/robots/drone/control_interface/ci_drone_flight_system_sensor.h

```diff
@@ -60,14 +84,18 @@ namespace argos {
 
    protected:
       CVector3 m_cPosition;
+      CVector3 m_cTargetPosition;
+      CVector3 m_cInitialPosition;
```

Comment:
I'm removing m_cInitialPosition, since I don't use it. 
Is it possible to just send a command to the Pixhawk to set it's initial position to zero before starting an experiment? 
I guess it is possible. Modifying the following code lines:   https://github.com/sinanOguz/argos3-srocs/blob/7491a79515242d11abb1b69e510c1ba5710f503d/src/plugins/robots/drone/hardware/drone_flight_system_default_sensor.cpp#L82-L89